.storybook/main.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  stories: ['../src/**/*.stories.js'],
+  addons: [
+    '@storybook/preset-create-react-app',
+    '@storybook/addon-actions',
+    '@storybook/addon-links',
+  ],
+};
```

Comment:
QQ: did you rebased or merged with master? 

I'm asking this because I think these are old changes that are already in master, so in theory, we don't need to review it again because it already had their review process. In this PR we have 39 files changes that are your changes + old changes (that are already in master). Having this makes a bit difficult to make a proper review of the functionalities that you try to add to the project. With this format, the reviewer would have extra work (to go by each commit to check what are truly your changes and review them).

I prefer to work with rebase, for this kind of situation (but this is just my bias opinion). I hope I'm not annoying and I hope this helps.

Let's check with @koonpeng .

Adding some helpful links:
https://www.atlassian.com/git/tutorials/merging-vs-rebasing
https://medium.com/datadriveninvestor/git-rebase-vs-merge-cc5199edd77c#:~:text=When%20you%20do%20rebase%20a,feature%20branch%20history%20remains%20same.

Reply:
I used `git pull --rebase` on this branch! was facing the situation where "`branch is x commits ahead and y commits behind master`" and then I was supposed to force push to overwrite the old changes but I think I used the wrong syntax then it ended up pushing all the commits without overwriting. I apologise for that.. ):

Reply:
Seems that a bunch of commits after f0c7931 are duplicated, im not sure what causes that. @matiasbavera Do you have a suggestion to clean it up? I'm thinking of cherry-picking but I don't know if there is a better way.

Reply:
I pushed a temp branch `tmp/commands-panel`, @vallq please check if it contains all the newer changes.

Reply:
okay @koonpeng looking at it now!

Reply:
Yeah @koonpeng, I think cherry pick will be our best option here if we want a clean history. But I'm case we don't we can also checkout files from master

`git checkout master --  file.ts`

(Sorry I'm writing from my cellphone)

Reply:
Don't worry @vallq, people "learn by doing" and we are here to help =)